enhancements/dns/externaldns-operator.md

```diff
@@ -235,15 +233,19 @@ such as a secret or configmap in an arbitrary namespace.
 to run in concert, one for each ExternalDNS operator Custom Resource (similar to how the OpenShift [Ingress Operator](https://github.com/openshift/cluster-ingress-operator)
-consumes one or more Ingress Controller resources). The `domains` field would be used to filter source resources according to the requested DNS hostname
-(sans any domains matching any values specified in the `excludeDomains` field).
-If an ExternalDNS operator Custom Resource specifies multiple DNS Zones, multiple instances of ExternalDNS would be deployed within the same deployment accordingly.
+consumes one or more Ingress Controller resources). The `domains` field would be used to filter source resources based on the source resources desired hostname.
+The `domains` field should be expressive enough to allow a user to specify multiple sets of domains that are considered included or excluded.
+In the ExternalDNS operator CR above, ExternalDNS is configured to create DNS records with hostnames that fit under `my-cluster.my-domain.com` that are not under
+`apps.my-cluster.my-domain.con`.
+
+By default, ExternalDNS would create DNS records for all instances of the given Source resource. A union type would be defined to allow for
+source-specific parameters. For the example ExternalDNS CR above, the ExternalDNS operator would create DNS records for Route resources that have the annotation
+`some.annotation.io/foo=bar`, and would ignore the ExternalDNS custom Hostname Annotation (see [Implementation Details](#implementation-details) for more details about ExternalDNS
+annotations).
 
-By default, ExternalDNS would create DNS records for all instances of the given Source resources. A union type would be defined to allow for
-source-specific parameters. For the example ExternalDNS CR above, the ExternalDNS operator would create DNS records for Route resources that match the label (read: annotation, more on this later)
-`foo: bar`, and would ignore the ExternalDNS Host Target Override Annotation (see [Implementation Details](#implementation-details) for more details about ExternalDNS
-annotations). This example CR would also instruct ExternalDNS to create DNS records for CR instances of type `dnsrecords` (not a part of OCP 4.9 implementation).
-Finally, this example CR would instruct ExternalDNS to create DNS records in the given private zone for Service resource instances in namespace `foo`.
+Note that only one Source resource can be specified for each ExternalDNS operator CRD. This is to highlight the fact that a single ExternalDNS instance cannot manage more than one
+source resource, which means that one ExternalDNS deployment is required for each source resource. See [Upstream Issue 1961](https://github.com/kubernetes-sigs/external-dns/issues/1961)
+to better understand the inconvenient instance sprawl resultant of the design of ExternalDNS.
```

Comment:
"CRD" → "CR" (or "CRD instance").  Maybe we should shorten the last sentence.  * grin *.
```suggestion
Note that only one Source resource can be specified for each ExternalDNS operator CR. This is to highlight the fact that a single ExternalDNS instance cannot manage more than one
source resource, which means that one ExternalDNS deployment is required for each source resource. See [Upstream Issue 1961](https://github.com/kubernetes-sigs/external-dns/issues/1961)
to better understand this design limitation of ExternalDNS.
```

Reply:
fixed and fixed.